CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ list(APPEND CMAKE_MODULE_PATH
 include(pkg-utils)
 include(project-utils)
 
-
+option(snapshot-installer "Build snapshot installers, CI option requires snapshot-build-number variable during initial cmake call." OFF)
```

Comment:
I would do this a little differently: don't make this an option but initialize its value based on whether the build number was set or not. That way it's sufficient to specify the build number :point_up:

Reply:
> With regards to the needed environment variable, I think it might be an easier overall process if we dropped the need for an environment variable and use another cmake variable that can be set via command line instead. In fact we could even do this in a way that deprecates the need for the currently added option by simply checking whether that special variable is empty or not.
> 
> ```
> set(snapshot-build-number "0" CACHE STRING "<description>")
> ```
Then we can't really do this. A release build does not require this value. The WixSharp files add the trailing `.0` when the build number is not specified, because we are not using the format `x.x.x.x`. This is probably something we should have a real-time discussion about in a meeting or on IRC. Let me know which would be more convenient for everyone.

Reply:
All items discussed in the meeting have been pushed.